packages/vscode-extension/src/utilities/platform.ts

```diff
@@ -0,0 +1,21 @@
+import * as os from "os";
+
+export class Platform {
+  static OS: "macos" | "windows" | "unknown" = (() => {
+    const platform = os.platform();
+    switch (platform) {
+      case "darwin":
+        return "macos";
+      case "win32":
+        return "windows";
+      default:
+        return "macos";
+    }
+  })();
+
+  static select(obj: { macos: any; windows: any }) {
+    if (Platform.OS !== "unknown" && Platform.OS in obj) {
+      return obj[Platform.OS];
+    }
+  }
```

Comment:
If possible, use `unknown` instead of `any`. We don't have any fallback here so following implementation is equivalent:

```js
static select(options: { macos: unknown; windows: unknown }) {
  return options[Platform.OS];
}
```